portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx

```diff
@@ -699,14 +699,28 @@ const ListObjects = ({
   };
 
   const downloadObject = (object: BucketObject) => {
-    if (object.size > 104857600) {
-      // If file is bigger than 100MB we show a notification
-      setSnackBarMessage(
-        "Download process started, it may take a few moments to complete"
+    if (object.name.endsWith("/")) {
+      download(
+        bucketName,
+        encodeFileName(object.name),
+        object.version_id,
+        true
       );
-    }
+    } else {
+      if (object.size > 104857600) {
+        // If file is bigger than 100MB we show a notification
+        setSnackBarMessage(
+          "Download process started, it may take a few moments to complete"
+        );
+      }
 
-    download(bucketName, encodeFileName(object.name), object.version_id);
+      download(
+        bucketName,
+        encodeFileName(object.name),
+        object.version_id,
+        false
+      );
+    }
```

Comment:
This can be rewritten as follows:

```suggestion
    if (object.size > 104857600) {
        // If file is bigger than 100MB we show a notification
        setSnackBarMessage(
          "Download process started, it may take a few moments to complete"
        );
      }

      download(
        bucketName,
        encodeFileName(object.name),
        object.version_id,
        object.name.endsWith("/")
      );
```

Reply:
fixed